base/util_test.go

```diff
@@ -638,31 +646,35 @@ func TestSetUpTestLogging(t *testing.T) {
 	require.Equal(t, LevelInfo, *consoleLogger.LogLevel)
 	require.Equal(t, *logKeyMask(KeyHTTP), *consoleLogger.LogKeyMask)
 
-	teardownFn := SetUpTestLogging(LevelDebug, KeyDCP, KeySync)
+	mock := &mockTB{}
```

Comment:
I can see why this was required for testing `SetUpTestLogging`, but I think tweaking the test to be running `setTestLogging` and executing the returned function would be sufficient.